### PR TITLE
Add and edit tables for multiplayer

### PR DESCRIPTION
**Pull request by andreacx**
# Changes :gem:
_Fix tables and functions for multiplayer_
* Add PlayerInGame table
* Edit Games table
* Add functions for manipulating PlayerInGame table
* Change functionality in api for talking to db
* Fix test for new tables

# Note
_Add code for deleting all tables before creating_

### CODE REVIEW

File: src/test/test_api.py [hunk 1]
```diff
@@ -223,18 +224,30 @@ def construct_path(dir_list):
...
         "total":[{"name":"ole","score":105},{"name":"mari","score":83}]}
     """
+
     # get response
     res = client.get("/viewHighScore")
     response = json.loads(res.data)
     #check that data structure is correct
+    if response["total"][0] is None:
+        # write to database to make sure there is something to read
+        write_to_db()
```
Comment (joaaax) [inline src/test/test_api.py:241]: Add newline after statement

File: src/webapp/models.py [hunk 2]
```diff
@@ -128,53 +169,56 @@ def update_game(token, session_num, play_time):
...
     """
     try:
-        db.session.query(Games).filter(Games.date < (datetime.datetime.today()
-                                                     - datetime.timedelta(hours=1))).delete()
+        games = db.session.query(Games).filter(Games.date < (
+            datetime.datetime.today() - datetime.timedelta(hours=1))).all()
+        for game in games:
+            db.session.query(PlayerInGame).filter(
+                PlayerInGame.gid == game.gid).delete()
+            db.session.delete(game)
```
Comment (joaaax) [inline src/webapp/models.py:216]: Add newline after loop

File: src/webapp/models.py [hunk 3]
```diff
@@ -10,24 +10,16 @@
...
-
 class Games(db.Model):
     """
        This is the Games model in the database. It is important that the
        inserted values match the column values. Token column value cannot
        be String when a long hex is given.
     """
 
-    token = db.Column(db.NVARCHAR(32), primary_key=True)
+    gid = db.Column(db.NVARCHAR(32), primary_key=True)
```
Comment (mahbx) [inline src/webapp/models.py:20]: vår konvensjon er vel g_id eller game_id?

File: src/webapp/models.py [hunk 4]
```diff
@@ -37,46 +29,56 @@ class Scores(db.Model):
         inserted values match the column values.
     """
 
-    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    sid = db.Column(db.Integer, primary_key=True, autoincrement=True)
```
Comment (mahbx) [inline src/webapp/models.py:32]: s_id eller score_id

**Pull request merged**